bin/build-bundles.sh

```diff
@@ -1,27 +0,0 @@
-#! /bin/bash
-
-function build_bundle {
-    if [ -f "Gruntfile.js" ]
-    then
-        grunt build
-        OUT=$? # return code of grunt build
-
-        if [ OUT != 0 ];
-        then
-            npm install
-            rm -rf Resources/public/dist
-            grunt build
-        fi
-    fi
-}
-
-cd src/Sulu/Bundle
-
-for bundle in *; do
-    if [ -d "$bundle" ];
-    then
-        cd "$bundle"
-        build_bundle
-        cd ..
-    fi
-done
```

Comment:
This file should not be removed, it is very handy to compile all the js/css stuff for all the bundles.